sycl/include/sycl/accessor.hpp

```diff
@@ -1793,6 +1794,49 @@ class __SYCL_SPECIAL_CLASS accessor :
     return constant_ptr<DataT>(getPointerAdjusted());
   }
 
+  // accessor::has_property for runtime properties is only available in host
+  // code
+  template <typename Property>
+  typename sycl::detail::enable_if_t<
+      !ext::oneapi::is_compile_time_property<Property>::value, bool>
+  has_property() const {
```

Comment:
There's a little bit of a conflict here, but I think this should be `noexcept`, despite [SYCL_ONEAPI_accessor_properties](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/supported/sycl_ext_oneapi_accessor_properties.asciidoc) not saying it is. @gmlueck - Do you agree?

Reply:
Added in fb9beca0b25db3b31f1f975f522fddf205e9330f

Reply:
Yes, makes sense.  Will someone update the spec?

Reply:
@steffenlarsen @gmlueck is it enough https://github.com/intel/llvm/pull/6632 ?

Reply:
I believe so. Thanks for fixing it! 😄